ioctl/cmd/account/accountcreate2.go

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	"github.com/tyler-smith/go-bip39"
+
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/pkg/util/fileutil"
+	hdwallet "github.com/miguelmota/go-ethereum-hdwallet"
+
+	ecrypto "github.com/ethereum/go-ethereum/crypto"
+)
+
+// Multi-language support
+var (
+	createByMnemonicCmdShorts = map[config.Language]string{
+		config.English: "Create new account for ioctl by mnemonic",
+		config.Chinese: "为ioctl创建新账户通过助记词",
+	}
+	createByMnemonicCmdUses = map[config.Language]string{
+		config.English: "create2",
+		config.Chinese: "create2 创建",
+	}
+
+	mnemonicConfigFilePath = filepath.Join(config.ReadConfig.Wallet, "mnemonic.yml")
+)
+
+// accountCreateCmd represents the account create command
+var accountCreateByMnemonicCmd = &cobra.Command{
+	Use:   config.TranslateInLang(createByMnemonicCmdUses, config.UILanguage),
+	Short: config.TranslateInLang(createByMnemonicCmdShorts, config.UILanguage),
+	Args:  cobra.ExactArgs(0),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := accountCreateByMnemonic()
+		return output.PrintError(err)
+	},
+}
+
+type mnemonicConfig struct {
+	PrivateKey []byte `yaml:"k"`
```

Comment:
better name this `encryptionKey`? it is used to encrypt the mnemonic and password